### PR TITLE
Feature/show crud

### PR DESCRIPTION
**Pull request by marcel-strzalka**
This PR adds:

- Show CRUD. Every show has a start time and a movie and hall belongs to it.
- validation for various Show attributes. For example you cannot have two different shows in the same hall at the same time.
- support for local time zone.
- some necessary updates to Movie and Hall models.
- seeds for Shows.
- different Show views with authentication and authorization support.

### CODE REVIEW

File: app/controllers/shows_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  before_action except: %i[index show] do
+    redirect_to movies_path unless manager?
+  end
+  before_action :find_show, only: %i[show destroy]
+
+  def index
+    @movies = Movie.all
+  end
+
+  def show; end
```
Comment (RubyDiver) [inline app/controllers/shows_controller.rb:13]: This line to fix

Comment (RubyDiver) [inline app/controllers/shows_controller.rb:13]: Line 13 need to be fixed

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,4 +1,7 @@
 # frozen_string_literal: true
 
 module ApplicationHelper
+  def pretty_time(datetime)
```
Comment (RubyDiver) [inline app/helpers/application_helper.rb:4]: Think of a more suitable name

File: app/helpers/shows_helper.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
```
Comment (RubyDiver) [inline app/helpers/shows_helper.rb:1]: Do you need this empty file?

Comment (h4ppyr0gu3) [reply app/helpers/shows_helper.rb:1]: +1

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (h4ppyr0gu3) [reply app/helpers/application_helper.rb:4]: maybe formatted_time would be better here

File: app/controllers/shows_controller.rb (hunk 1 above)
Comment (h4ppyr0gu3) [reply app/controllers/shows_controller.rb:13]: this is fine unless the endpoint is unused then it could be deleted

**Review by h4ppyr0gu3 — APPROVED** [review]

File: app/controllers/shows_controller.rb (hunk 1 above)
Comment (marcel-strzalka) [reply app/controllers/shows_controller.rb:13]: This endpoint is used, so I won't delete it. I got that from an Rubocop autocorrection.

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (marcel-strzalka) [reply app/helpers/application_helper.rb:4]: Sure, gonna use formatted_time instead of pretty_time.

**Pull request merged**